### ISSUE TITLE
Error in Example Demo

### ISSUE THREAD

**Issue by dikaio** [nextauthjs/next-auth#1049]
The example demo currently produces an [application error here](https://next-auth-example.now.sh/protected-ssr) while signed in. You need to be signed in to see the error.

**Comment by balazsorban44** [comment]
Could you please specify what kind of error you are seeing, and how to reproduce it?

**Comment by dikaio** [comment]
Hi @balazsorban44. Sure.

Your example application `https://github.com/nextauthjs/next-auth-example` produces an application error on the `protected-ssr` page, [you can see the error here](https://next-auth-example.now.sh/protected-ssr).

To reproduce the error:

```
visit https://next-auth-example.now.sh/
sign in
visit https://next-auth-example.now.sh/protected-ssr
```

This will produce the error

```
502: BAD_GATEWAY
Code: NO_RESPONSE_FROM_FUNCTION
ID: sfo1::dpn2b-1609986547622-a9ca018084fa`
```

This error is not produced from cloning and using your repo, its the example application you guys have hosted.

**Comment by balazsorban44** [comment]
Thanks! I am wondering if these are related:
#660, #1021

**Comment by iaincollins** [comment]
Thanks for raising this! I saw it earlier this week and came here to check to see if anyone had flagged it.

The demo actually uses an in memory database, so has no external dependencies (beyond OAuth provides).

The Protected SSR route and the JWT route are both not working, but were previously.

I think there may have been some sort of change on the Vercel platform that is triggering this issue.

**Comment by iaincollins** [comment]
I'm happy to pick this up and debug it - that's probably the best idea as I'm the only one who currently has access to the logs or to do deployments, as we don't have paid or open source team account yet.

I checked the logs and do have an error but it didn't seem helpful, so will likely create a custom page with some debugging. 

It's seems likely the problem is with with decoding the JWT somehow, as both the JWT endpoint and the Protected endpoint use it. I have no idea why that would fail but it should be easy to debug and replicate by creating a simple API route that handles decoding the token and trapping any errors it throws.

**Comment by iaincollins** [comment]
Okay nope, some incorrect breaking changes were made to the repo I was unaware of. I'm going to ask folks that not make changes the Example repo for now.

**Comment by iaincollins** [comment]
Okies:

- I've rolled out a fix for the JWT page and removed the SSR example entirely (it wasn't intended to be part of the live demo).
- I've updated the example to Next.js 10.
- I've closed the open PRs, and explained why in the case of the VERCEL_URL PR.

I think we should leave the exiting example repo alone for now.

Hopefully in the future we can have a wider range of examples (with appropriate tests and pre-live environments in place!)
